spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -104,6 +104,14 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(generateBools(), DataTypes.IntegerType)
   }
 
+  test("cast double to string") {
```

Comment:
for consistency with other methods:

```suggestion
  test("cast DoubleType to StringType") {
```

Reply:
I saw that there were already defined functions for both types so I moved them there